loan-events/src/agent.spec.js

```diff
@@ -0,0 +1,255 @@
+const mockContract = {};
+
+jest.mock('forta-agent', () => ({
+  ...jest.requireActual('forta-agent'),
+  getEthersProvider: jest.fn(),
+  ethers: {
+    ...jest.requireActual('ethers'),
+    Contract: jest.fn().mockReturnValue(mockContract),
+  },
+}));
+
+const {
+  Finding, FindingType, FindingSeverity, ethers, TransactionEvent,
+} = require('forta-agent');
+
+const { provideHandleTransaction, provideInitialize } = require('./agent');
+const {
+  getObjectsFromAbi,
+  getEventFromConfig,
+  createMockEventLogs,
+} = require('./test-utils');
+
+const utils = require('./utils');
+
+const config = require('../bot-config.json');
+
+// utility function specific for this test module
+// we are intentionally not using the Forta SDK function due to issues with
+// jest mocking the module and interfering with default function values
+function createTransactionEvent(txObject) {
+  const txEvent = new TransactionEvent(
+    null,
+    null,
+    null,
+    [],
+    {},
+    null,
+    txObject.logs,
+    null,
+  );
+  return txEvent;
+}
+
+// check the configuration file to verify the values
+describe('check bot configuration file', () => {
+  it('procotolName key required', () => {
```

Comment:
```suggestion
  it('protocolName key required', () => {
```
Copypasta typo. It's everywhere.